app/retail/emails.py

```diff
@@ -1092,6 +1099,24 @@ def bounty_feedback(request):
     return HttpResponse(response_html)
 
 
+@staff_member_required
+def funder_payout_reminder(request):
+    """Display the stale funder email template.
+
+    Params:
+        limit (int): The number of days to limit the scope of the email to.
+        duration_copy (str): The copy to use for associated duration text.
+        username (str): The Github username to reference in the email.
+
+    Returns:
+        HttpResponse: The HTML version of the templated HTTP response.
+
+    """
+    bounty = Bounty.objects.filter(fulfillment_submitted_on__isnull=False).first()
+    github_username = request.GET.get('username', '@foo')
+    response_html, _ = render_funder_payout_reminder(username, limit, duration_copy)
+    return HttpResponse(response_html)
```

Comment:
^ needs to be re-written ! Doesn't work at the moment aka the admin view for the email